### PR TITLE
feat(frontend): add meta to fiche service public and ministere travail

### PR DESCRIPTION
**Pull request by lionelB**
add meta description to fiche-service-public and fiche ministere-travail

### CODE REVIEW

**Review by UnbearableBear — APPROVED** [review]
👍

File: packages/code-du-travail-frontend/pages/fiche-ministere-travail.js [hunk 1]
```diff
@@ -42,6 +43,9 @@ class Fiche extends React.Component {
     );
     return (
       <PageLayout>
+        <Head>
+          <meta name="description" content={data._source.description} />
```
Comment (douglasduteil) [inline packages/code-du-travail-frontend/pages/fiche-ministere-travail.js:47]: Duplicate with #771

File: packages/code-du-travail-frontend/pages/fiche-service-public.js [hunk 2]
```diff
@@ -49,6 +50,9 @@ class Fiche extends React.Component {
     const footer = <Source name="service-public.fr" url={data._source.url} />;
     return (
       <PageLayout>
+        <Head>
+          <meta name="description" content={data._source.description} />
```
Comment (douglasduteil) [inline packages/code-du-travail-frontend/pages/fiche-service-public.js:54]: Duplicate with #771

**Review by douglasduteil — CHANGES_REQUESTED** [review]
- meta tag are out of topic here
- This is more a `feat(dataset)` then `feat(frontend)`

File: packages/code-du-travail-frontend/pages/fiche-ministere-travail.js (hunk 1 above)
Comment (lionelB) [reply packages/code-du-travail-frontend/pages/fiche-ministere-travail.js:47]: not the same page

File: packages/code-du-travail-frontend/pages/fiche-service-public.js (hunk 2 above)
Comment (lionelB) [reply packages/code-du-travail-frontend/pages/fiche-service-public.js:54]: not the same page

**Review by douglasduteil — APPROVED** [review]
Alright

**Pull request merged**